SegM8/example-clock/patch.xodp

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
Here is an Idea of a true clock emulation using XOD cloud and MCU Tethering INET.
![3](https://user-images.githubusercontent.com/35202630/91069334-f2da2e80-e63d-11ea-8b86-4877bbc8607f.png)

Reply:
Can you wait until these features are in the XOD release? "Cloud / Basics" is not there yet.

Reply:
done